Library/models.py

```diff
@@ -73,3 +73,20 @@ def is_staff(self):
         # Simplest possible answer: All admins are staff
         return self.is_admin
 
+
+class Books(models.Model):
+    id = models.AutoField(primary_key=True)
+    title = models.CharField(max_length=255)
+    author = models.CharField(max_length=255)
+    isbn = models.CharField(max_length=13, unique=True)
+    publisher = models.CharField(max_length=255)
+    publication_date = models.DateField()
+    genre = models.CharField(max_length=100)

```

Comment:
genre field should be restricted, as we have few genre that is present for books. That should be introduced that the column can contain the only the field that is an authentic genre